BootloaderCommonPkg/Library/IppCryptoPerfLib/IppCryptoPerfLib.c

```diff
@@ -0,0 +1,137 @@
+
+/* @file
+*
+*  Copyright (c) 2024, Intel Corporation. All rights reserved.<BR>
+*
+*  SPDX-License-Identifier: BSD-2-Clause-Patent
+*
+*/
+
+#include <Library/CryptoLib.h>
+#include <Library/ShellLib.h>
+#include <Library/TimeStampLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/SecureBootLib.h>
+
+
+#define KB_1   0x0400
+#define KB_8   0x2000
+#define KB_32  0x8000
+#define KB_64  0x10000
+#define KB_128 0x20000
+#define KB_256 0x40000
+#define MB_1   0x00100000
+#define MB_2   0x00200000
+#define MB_4   0x00400000
+#define MB_8   0x00800000
+#define MB_10  0x00A00000
+#define MB_16  0x01000000
```

Comment:
Base.h already defined some size as below. please use the existing one.
#define  SIZE_1KB    0x00000400
#define  SIZE_2KB    0x00000800
#define  SIZE_4KB    0x00001000
#define  SIZE_8KB    0x00002000
#define  SIZE_16KB   0x00004000
#define  SIZE_32KB   0x00008000
#define  SIZE_64KB   0x00010000
#define  SIZE_128KB  0x00020000
#define  SIZE_256KB  0x00040000
#define  SIZE_512KB  0x00080000
#define  SIZE_1MB    0x00100000
#define  SIZE_2MB    0x00200000
#define  SIZE_4MB    0x00400000
#define  SIZE_8MB    0x00800000
#define  SIZE_16MB   0x01000000
#define  SIZE_32MB   0x02000000
#define  SIZE_64MB   0x04000000
#define  SIZE_128MB  0x08000000
#define  SIZE_256MB  0x10000000
#define  SIZE_512MB  0x20000000
#define  SIZE_1GB    0x40000000
#define  SIZE_2GB    0x80000000